spec/models/stripe_wrapper_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "spec_helper"
+
+describe StripeWrapper do
+  describe StripeWrapper::Charge do
+    describe ".create" do
+      it "makes a successful charge", :vcr do
```

Comment:
`:vcr` tag could be refactored to the level higher, as it is @ every spec anyway.